src/adic_space.lean

```diff
@@ -76,16 +80,16 @@ A morphism in 𝒞 is a map of top spaces, an f-map of presheaves, such that the
 map on the stalks pulls one valuation back to the other.
 -/
 
-instance presheaf_of_rings.comm_ring {X : Type*} [topological_space X]
+instance presheaf_of_rings.comm_ring {X : Type u} [topological_space X]
```

Comment:
Why is all this in `adic_space.lean`? I feel like this file should be split a lot

Reply:
Let's do that in a follow-up PR.